code/modules/medical/genetics/geneticsResearch.dm

```diff
@@ -311,6 +311,21 @@ var/datum/geneticsResearchManager/genResearch = new()
 		genResearch.max_material += 50
 		..()
 
+/datum/geneticsResearchEntry/gene_booth_speedup_complex
+	name = "Complex Gene Booth Acceleration"
+	desc = "Increases the working speed of the Gene Booth by an additional +25%."
+	researchTime = 1200
+	researchCost = 150
+	tier = 2
+	requiredMutRes = list("early_secret_access")
+	requiredResearch = list(/datum/geneticsResearchEntry/genebooth)
+
+	onFinish()
+		var/obj/machinery/genetics_booth/type = /obj/machinery/genetics_booth
+		type.process_speedup += 0.25
+		. = type // we get warned about type being defined but unused and idk how else to make byond shut up
```

Comment:
```suggestion
		type = type // we get warned about type being defined but unused and idk how else to make byond shut up
```
We'll be able to suppress warnings in 515. Until then this is a bit cleaner since it removes the warning while accomplishing absolutely nothing.